fvtest/tril/tril/simple_compiler.hpp

```diff
@@ -29,15 +29,15 @@ namespace TR { class IlVerifier; }
 namespace Tril {
 
 /**
- * @brief Concrete realization of MethodCompiler that uses JitBuilder for compilation
+ * @brief Concrete realization of MethodCompiler 
  */
-class JitBuilderCompiler : public Tril::MethodCompiler {
+class SimpleCompiler : public Tril::MethodCompiler {
```

Comment:
Perhaps I'm being obtuse, but can you explain what's wrong with calling this a JitBuilderCompiler?  Is it because a "SimpleCompiler" is just one kind of an instance of a JitBuilder-based compiler?  I'm just trying to connect the dots...

Reply:
The distinction is that this is a concrete compiler implementation that can be satisfied by both the test compiler and JitBuilder backends -- it's not directly related to JitBuilder anymore.